### ISSUE TITLE
Incorrect line wrapping in grid row with .col-xs-1, .col-xs-11 in Firefox with Flexbox enabled

### ISSUE THREAD

**Issue by stephaniepurvis** [twbs/bootstrap#18420]
I have a codepen demonstrating this problem here: http://codepen.io/stephaniekendall/pen/XXWJBx

At 585px and below in Firefox (v. 42.0) for Mac, these columns collapse into a single column with the divs stacked on top of each other. The columns behave as expected for me in both Chrome and Safari.  It only seems to be a problem when you have 1 col next to 11 cols.

**Comment by twbs-lmvtfy** [comment]
Hi @stephaniepurvis!

You appear to have posted a live example (http://codepen.io/stephaniekendall/pen/XXWJBx.html), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 8, column 1: [E003](https://github.com/twbs/bootlint/wiki/E003): Found one or more `.row`s that were not children of a grid column or descendants of a `.container` or `.container-fluid`

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by stephaniepurvis** [comment]
I added a .container-fluid div around the example code per twbs-lmvtfy's message above. The error isn't one that makes a difference to the issue though.

**Comment by cvrebert** [comment]
@stephaniepurvis Looks like you're using the Flexbox version of the grid. Is that correct?

**Comment by cvrebert** [comment]
Possible duplicate of https://github.com/twbs/bootstrap/issues/17310#issuecomment-152995846

**Comment by stephaniepurvis** [comment]
Yes, that is correct. I noticed this while playing with the flexbox grid being enabled. I will check tomorrow to see if the same thing happens without the flexbox grid.

**Comment by cvrebert** [comment]
Confirmed in OS X Firefox 44.0a2 (2015-12-03) at viewport widths <=615px.

**Comment by cvrebert** [comment]
@stephaniepurvis Your CodePen seems to have been deleted. Do you have another copy or could you recreate the code?
I made http://jsbin.com/nuwanet/edit?html,css just based on the issue description, but can't seem to reproduce in Firefox 48.0a2.

**Comment by twbs-lmvtfy** [comment]
Hi @cvrebert!

You appear to have posted a live example (http://jsbin.com/nuwanet/edit), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* [W002](https://github.com/twbs/bootlint/wiki/W002): `<head>` is missing X-UA-Compatible `<meta>` tag that disables old IE compatibility modes

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by stephaniepurvis** [comment]
Sorry about my code pen being gone. If I deleted it, it was on accident!  I tried recreating it here: http://codepen.io/stephaniekendall/pen/zBqrvM

I know it was only happening with flexbox enabled, so I am using the compiled version of the css with flexbox = true in my code pen. My version of Firefox is now at 47.0 and I don't seem to be encountering the problem anymore. Since it was only happening in FF, I wonder if it was a bug with FF version 42?
